src/Quartz/Core/QuartzScheduler.cs

```diff
@@ -1265,6 +1266,7 @@ public virtual Task<IJobDetail> GetJobDetail(JobKey jobKey)
 
             return await resources.JobStore.RetrieveTrigger(triggerKey).ConfigureAwait(false);
```

Comment:
Yeah sorry I should have linked it. It is pretty handy. Especially of you do not know the drill yet.